src/components/AttachmentsButton/AttachmentsButton.tsx

```diff
@@ -59,16 +62,19 @@ export const AttachmentsButton = () => {
           value: URL.createObjectURL(file),
           file: file
         })
+        trackEvent('hl_chat_image_attachment_added', { fileType: file.type });
       } else if (file.type === 'application/pdf') {
         addAttachment({
           type: 'pdf',
           value: file
         })
+        trackEvent('hl_chat_pdf_attachment_added', {});
       } else if (file.type === 'text/csv' || file.type === 'application/vnd.ms-excel' || file.type === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
         addAttachment({
           type: 'spreadsheet',
           value: file
         })
+        trackEvent('hl_chat_spreadsheet_attachment_added', { fileType: file.type });
```

Comment:
For events which are all essentially performing the same or very similar actions, we can do something like this:

```
trackEvent('Chat Attachment Added', { type: 'audio' })
trackEvent('Chat Attachment Added', { type: 'pdf' })
trackEvent('Chat Attachment Added', { type: 'spreadsheet' })
```

Reply:
Same with the permission denied events below (could use `Chat Permission Denied`), and clipboard image and text.

Reply:
Done